_data/simple-icons.json

```
@@ -8935,6 +8935,12 @@
             "source": "https://brand.netflix.com/en/assets/brand-symbol",
             "guidelines": "https://brand.netflix.com/en/assets/brand-symbol"
         },
+        {
+            "title": "Netgear",
```

```suggestion
            "title": "NETGEAR",
```
They always capitalize the name.